global/wizard.sh

```diff
@@ -1,26 +1,27 @@
 #!/bin/bash
 DIR=$(pwd)
 CLANG=${CLANG:-C.UTF-8}
-IMAGE=${IMAGE:-netways/showoff:0.19.6}
+IMAGE=${IMAGE:-netways/showoff:0.20.4}
 CNAME=${CNAME:-showoff}
 TRAINING=${TRAINING:-$(basename "$DIR")}
-DOCKER=${DOCKER:-$(command -v docker)}
+RUNTIME=${RUNTIME:-$(command -v docker)}
 GIT=${GIT:-$(command -v git)}
 SCRIPT_DIR=$(cd $(dirname $0); pwd -P)
 NO_RESET=${NO_RESET:-""}
 
-# Functions
-
+# Run given command in the showoff container
 execdocker () {
-  if [[ -n $($DOCKER ps -aq -f name=$CNAME 2> /dev/null) ]]; then
-    $DOCKER rm -f $CNAME 2> /dev/null
+  if [[ -n $($RUNTIME ps -aq -f name=$CNAME 2> /dev/null) ]]; then
+    $RUNTIME rm -f $CNAME 2> /dev/null
   fi
 
-  $DOCKER run \
+  $RUNTIME run \
     -it \
     --name=$CNAME \
     --rm \
     -p 9090:9090 \
+    --user $(id -u) \
+    --userns=keep-id \
```

Comment:
:thinking: needs to be optional for Docker

Reply:
It only works if you explicitly set the Environment variable $RUNTIME as the command -v enforces docker which works perfectly fine with an alias.  The alias is also fine if we switch the case statement to `case $($RUNTIME -v) in`. (tested with podman only which shows "podman version X.Y.Z", but docker has "Docker version X.Y.Z, build HASH" as compact version string so it should be fine)

Reply:
I usually use it like this, which works in both cases

```
RUNTIME=$(command -v docker) ./global/wizard.sh

RUNTIME=$(command -v podman) ./global/wizard.sh
```

Reply:
@dgoetz Is the current version OK you think?

Reply:
Updated the Branch